packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -70,27 +70,34 @@ interface ModalWizardProps {
     /**
      * Props to pass to each modal
      */
-    modalProps?: any;
+    modalProps?: Partial<ModalProps>;
+
+    /**
+     * Classname for modal
+     */
+    modalClassName?: any;
 
     /**
      * Label for close button
      */
     closeButtonLabel?: string;
 
+    /**
+     * Props for progress bar
+     */
+    progressBarProps?: Partial<ProgressProps>;
```

Comment:
```suggestion
    progressBarProps?: Omit<ProgressProps, 'value'>;
```

Reply:
After hard-coding the style of progressBar, I removed this prop here